### PR TITLE
Develop contacts list

### PR DESCRIPTION
**Pull request by Whitney-Rene**
This branch develop a contacts web application.
This adds code and functionality that will display all contacts to the user and allow the user to add new contacts and update existing contacts.
There is code for css as well.

### CODE REVIEW

File: client/src/components/create-contacts.jsx
Comment (plee-fn) [inline client/src/components/create-contacts.jsx:1]: Usually, component names are pascal case. Meaning, the first letter of each word should be capitalized. Consider changing `create-contacts.jsx` to `CreateContacts.jsx`. I also think adding what the component is would be helpful in identifying it (i.e.`CreateContactsForm.jsx`)

File: client/src/components/edit-contacts.jsx
Comment (plee-fn) [inline client/src/components/edit-contacts.jsx:1]: Same comment I made for `create-contacts.jsx` about pascal case. Considering updating the file name to `EditContactsModal.jsx`

File: client/src/components/view-contacts.jsx
Comment (plee-fn) [inline client/src/components/view-contacts.jsx:1]: Same comment about pascal case from above. Consider updating file name to `ViewContacts.jsx`

File: client/src/components/contacts.jsx
Comment (plee-fn) [inline client/src/components/contacts.jsx:1]: Consider changing the file name to `Contacts.jsx` as pascal case is usually the best practice for the react component naming scheme.

File: client/src/app.test.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import { describe, expect, test, vitest } from 'vitest';
```
Comment (plee-fn) [inline client/src/app.test.jsx:2]: I'm not too familiar with vitest... but if its the same as jest. We can definitely go over how to do some RTL tests. Let me know.

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    const values = [
+        updatedContact.name,
+        updatedContact.email,
+        updatedContact.phonenumber,
+        updatedContact.notes
+    ];
+
+    try {
+         const updated = await db.query(query, values);
+         console.log(updated.rows[0]);
```
Comment (plee-fn) [inline server/server.js:77]: don't forget to remove console.logs by either deleting or commenting it out. Leaving console.logs can sometimes make performance slower.

File: client/src/components/create-contacts.jsx
Comment (Whitney-Rene) [reply client/src/components/create-contacts.jsx:1]: Thank you Philip.  This very helpful feedback.  I will implement it for this project and the next!

File: client/src/components/view-contacts.jsx
Comment (Whitney-Rene) [reply client/src/components/view-contacts.jsx:1]: Thank you!

File: client/src/components/contacts.jsx
Comment (Whitney-Rene) [reply client/src/components/contacts.jsx:1]: Thank you!

File: client/src/app.test.jsx (hunk 1 above)
Comment (Whitney-Rene) [reply client/src/app.test.jsx:2]: I would LOVE to go over this! Thank you for offering Philip.  Yes, vitest is very similar to jest.

File: server/server.js (hunk 2 above)
Comment (Whitney-Rene) [reply server/server.js:77]: Thank you for this reminder Philip.

**Pull request merged**